languages/python.md

```diff
@@ -23,6 +23,37 @@ def widgetize(widget):
   standardize(widget)
 ```
 
+## Linting
+It is recommended to use [black](https://black.readthedocs.io/en/stable/) (formatting), [flake8](https://flake8.pycqa.org/en/latest/index.html#quickstart) (formatting), [bandit](https://bandit.readthedocs.io/en/latest/) (security), and [isort](https://pycqa.github.io/isort/) (import order) during development and CI. Black is preferred as the primary formatting style due to its easy methods for auto-formatting the code. 
+
+To install these linters when starting a new project, run `pipenv install black flake8 bandit isort`. The linters can be run together with `make lint` after adding the following commands to a project's Makefile:
+
+```
```

Comment:
These Make commands are specific to the llama application. Rather than including them all here, I would just link to the Makefile for the llama app as an example.

Reply:
Done!